package.json

```diff
@@ -43,6 +43,7 @@
     "line-awesome": "github:icons8/line-awesome",
     "lodash": "^4.17.20",
     "mini-css-extract-plugin": "0.5.0",
+    "mui-datatables": "^3.4.1",
```

Comment:
Can you use the bento-components data table

Reply:
fixed